CHANGELOG.md

```diff
@@ -4,6 +4,41 @@
 
 ## [Unreleased (beta)]
 
+## [0.13.0] - 2018-06-20
+
+Release notes: https://github.com/DataDog/dd-trace-rb/releases/tag/v0.13.0
+
+Git diff: https://github.com/DataDog/dd-trace-rb/compare/v0.12.1...v0.13.0
+
+### Added
+
+- Sequel integration (supporting Ruby 2.0+) (#171, #367) (@randy-girard, @twe4ked, @palin)
+- gRPC integration (supporting Ruby 2.2+) (#379, #403) (@Jared-Prime)
+- ActiveModelSerializers integration (#340) (@sullimander)
+- Excon integration (#211) (@walterking)
+- Rake integration (supporting Ruby 2.0+, Rake 12.0+) (#409)
+- Request queuing tracing to Rack (experimental) (#272)
+- ActiveSupport::Notifications::Event helper for event tracing (#400)
+- Request and response header tags to Rack (#389)
+- MySQL2 integration (#453) (@jamiehodge)
+- Sidekiq tag job delay (#458) (@gottfrois)
+- Sinatra tag request id (#427)
```

Comment:
I don't think we actually added a `request_id` tag at all. We added "Request and response headers to Sinatra". This also needs to list #375.

Reply:
We do tag request id, but its because its listed as one of the default headers we tag. Yeah but you're right the description needs to be expanded

Reply:
Well what we did for Sinatra is consistent with what we did for Rack. So the comment should probably look the same.